content/en/docs/ops/configuration/security/security-policy-examples/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Security policy examples
+description: Shows common examples of using Istio security policy.
+weight: 60
+owner: istio/wg-security-maintainers
+test: n/a
+---
+
+## Background
+
+This page shows common patterns of using Istio security policy. You may find them useful in your deployment or use this
+as a quick reference to example policies.
+
+The policy demonstrated here are just examples and please make necessary changes to adapt to your actual environment
+before applying.
+
+Also read the [authentication tasks](/docs/tasks/security/authentication/authn-policy) and the
+[authorization tasks](/docs/tasks/security/authorization) for hands-on tutorial of using the security policy with much
+more details.
+
+### Require mandatory JWT validation
+
+The `RequestAuthentication` policy defines a list of JWT issuers that are allowed, by default it also allows
+requests without JWT token.
+
+After defining the `RequestAuthentication` policy, use the following extra policy to enforce mandatory JWT validation
+that rejects the request if it has no JWT token:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: require-jwt
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: DENY
+  rules:
+  - from:
+    - source:
+        notRequestPrincipals: ["*"]
+EOF
+{{< /text >}}
+
+### Require different JWT issuer per host
+
+JWT validation is common on the ingress gateway and you may want to require different JWT issuers for different
+hosts, you can use the authorization policy for finer grained JWT validation in addition to the request authentication.
+
+Use the following policy if you want to allow requests only to the listed hosts and reject requests to any other hosts:
```

Comment:
Remove "only" and "reject...". Can you provide a more detailed description the the below policy? e.g. allow access to the given hosts if JWT principal matches.